cmd/kube-controller-manager/app/options/podgccontroller.go

```diff
@@ -33,7 +33,7 @@ func (o *PodGCControllerOptions) AddFlags(fs *pflag.FlagSet) {
 		return
 	}
 
-	fs.Int32Var(&o.TerminatedPodGCThreshold, "terminated-pod-gc-threshold", o.TerminatedPodGCThreshold, "Number of terminated pods that can exist before the terminated pod garbage collector starts deleting terminated pods. If <= 0, the terminated pod garbage collector is disabled.")
```

Comment:
thanks for the PR, but the default is already output in help text automatically:

```
kube-controller-manager --help | grep terminated-pod-gc-threshold
```

> --terminated-pod-gc-threshold int32   Number of terminated pods that can exist before the terminated pod garbage collector starts deleting terminated pods. If <= 0, the terminated pod garbage collector is disabled. (default 12500)